servers/quarkus-server/src/main/resources/application.properties

```diff
@@ -107,11 +107,12 @@ quarkus.resteasy.gzip.enabled=true
 ## Quarkus auth settings
 #quarkus.oidc.credentials.secret=
 #quarkus.oidc.client-id=
-#quarkus.oidc.auth-server-url=
+nessie.server.authentication.enabled=false
+# to be overwritten by end user when enabling OpenID validation
+quarkus.oidc.auth-server-url=http://127.255.0.0:0/auth/realms/unset/
```

Comment:
@snazy : I had to go back to a loopback address here, because using example addresses like `192.0.2.1` apparently adds about 10 sec to server startup time... I guess Quarkus tries to access it on startup, however, using a fake address like this does not break the startup.